packages/ui/src/mocks/data/mockMembers.ts

```diff
@@ -0,0 +1,11 @@
+import { MemberFieldsFragment } from '../../api/queries'
+import rawMembers from './raw/members.json'
+
+export type MockVideo = MemberFieldsFragment
```

Comment:
This name (`MockVideo`) does not seem right here. `MockMember`?

Reply:
Yep, my bad :+1: